packages/storybook/src/stories/ino-tab-bar/ino-tab-bar.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect, Locator, test } from '@playwright/test';
+import { goToStory } from '../test-utils';
+
+test.describe('ino-tab-bar', () => {
+  let inoTab: Locator;
+  let inoTabBar: Locator;
+
+  test.beforeEach(async ({ page }) => {
+    inoTabBar = page.locator('ino-tab-bar');
+    inoTab = inoTabBar.locator('ino-tab');
+  });
+
+  test('should render with the inoActiveTab property set correctly', async ({
+    page,
+  }) => {
+    await goToStory(page, ['Structure', 'ino-tab-bar', 'active-tab']);
+    await expect(inoTab.nth(1)).toHaveJSProperty('a11ySelected', true);
+  });
+
+  test('should be focused automatically with the autoFocus property', async ({
```

Comment:
We've tested this in our Stencil test, so there is no need to test it here again